frontend/lib/spree/frontend/middleware/seo_assist.rb

```diff
@@ -14,7 +14,8 @@ def call(env)
           taxon_id = params['taxon']
 
           # redirect requests using taxon id's to their permalinks
-          if !taxon_id.blank? && !taxon_id.is_a?(Hash) && taxon = Taxon.find(taxon_id)
+          taxon = Taxon.find(taxon_id)
+          if !taxon_id.blank? && !taxon_id.is_a?(Hash) && taxon
```

Comment:
This change introduces change in logic, since `find` will fail if taxon_id is blank and next line won't be called at all

Reply:
Ah, I understand. Okay I will undo.